contrib/firmware_packager/install_dell_bios_exe.py

```diff
@@ -81,6 +81,17 @@ def find_uefi_device(client, deviceid):
     sys.exit(1)
 
 
+def set_conf_only_trusted(client, setval):
+    prop = "OnlyTrusted"
+    curval = get_daemon_property(prop)
+    if curval is None:
+        print("Can't determine %s via D-Bus, please turn it off manually" % prop)
+    elif curval or setval:
+        modify_config(client, prop, str(setval).lower())
+        return get_daemon_property(prop) == setval
```

Comment:
I think the daemon needs to be restarted after making this change doesn't it?  So it shouldn't just read that new value immediately.

Reply:
With some tests here around `./dist/fwupd.sh` and `./contrib/firmware_packager/install_dell_bios_exe.py`, the property change gets emitted and notified upon writing to `./dist/etc/fwupd/daemon.conf` completed, it seems the daemon stats the configuration file therefore changes will takes effect immediately.